### ISSUE TITLE
Reception Desk / A way to leave messages.

### ISSUE THREAD

**Issue by Uraxys** [StrangeLoopGames/EcoSuggestions#1729]
It would be nice to have a feature that allows people to send messages to the government / people with elected titles.
This could for example be a reception desk, when the player interacts with the desk they are given the ability to write a message that the owner can read.

The only work around I've seen so far is either a 3rd party service like discord, or signs.

This could be useful for shops, where people could leave a message of what they wish the shop had, or to contact the government with road requests for example.

**Comment by CraigWCooper** [comment]
Can't you just DM them using the in game direct message system? They will then receive the message when they get back online.
I do like the idea of a Forum object. or even just another option on the contract board to leave Pinned messages.

**Comment by Uraxys** [comment]
> Can't you just DM them using the in game direct message system? They will then receive the message when they get back online. I do like the idea of a Forum object. or even just another option on the contract board to leave Pinned messages.

That is one of the options I've tried before, but it becomes a mess if the person that should receive the message is an elected official that can be replaced every time a new election is started.

**Comment by CraigWCooper** [comment]
> > Can't you just DM them using the in game direct message system? They will then receive the message when they get back online. I do like the idea of a Forum object. or even just another option on the contract board to leave Pinned messages.
> 
> That is one of the options I've tried before, but it becomes a mess if the person that should receive the message is an elected official that can be replaced every time a new election is started.

Yeah makes sense but tbf keeping up with political matter in eco is semi part of the game you should be involved in the elections of these officials thus knowing who they are. There shouldn't be a situation where you don't know who your official are unless you are really not caring in the first place.

Maybe an option to message 'titles' might be fun.
